recipes/kmedoids/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "kmedoids" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kno10/python-kmedoids/archive/refs/tags/v{{ version }}.tar.gz
+  #  `openssl sha256 <file name>`.
+  sha256: d334848ecc0d4b17c0e041b975f29b1963cd64e7c3743d9b9ec157ae5780e2c2
+
+build:
+  number: 0
+  script:
+    - export CARGO_NET_GIT_FETCH_WITH_CLI=true
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+    - numpy
+  host:
+    - python
+    - pip
+    - maturin
+  run:
+    - python
+    - numpy
+    - libgcc-ng  # [linux]
+
+test:
+  imports:
+    - kmedoids
+    - kmedoids.kmedoids
+  requires:
+    - python
+    - pip
+    - pytest
+    - numpy
```

```suggestion
    - pip
    - pytest
```

numpy and python are dependencies through the `run` section, no need to relist them,